charts/cluster/templates/NOTES.txt

```diff
@@ -0,0 +1,13 @@
+The {{ include "cluster.fullname" . }} has been installed successfully.
+
```

Comment:
maybe a summary of a few main configurations of the cluster could be helpful, nr of instances, image being used, backup configured or not (hinting at the fact that it should in production), schedules for backups and the backup destination, etc??

Reply:
That's a really good point. I'm on it.

Reply:
![image](https://github.com/cloudnative-pg/charts/assets/2123767/5890321b-1290-4033-affd-16ed38dee805)

Reply:
This looks awesome! 🎉